packages/p2p/src/components/app-content.jsx

```diff
@@ -76,7 +78,7 @@ const AppContent = ({ order_id }) => {
                 <TemporarilyBarredHint />
                 <BuySell />
             </div>
-            <div label={localize('Orders')}>
+            <div data-count={notification_count} label={localize('Orders')}>
```

Comment:
does this not give an eslint error?

Reply:
@ameerul-deriv, Nope, data-* attributes are valid html attributes. This is how we add custom attributes to` HTML` elements 😎